docs/CLI.md

```diff
@@ -16,14 +16,17 @@ WebdriverIO CLI runner
 Usage: wdio [options] [configFile]
 Usage: wdio config
 Usage: wdio repl <browserName>
+Usage: wdio install <type> <name>
 
 config file defaults to wdio.conf.js
 The [options] object will override values from the config file.
-An optional list of spec files can be piped to wdio that will override
-configured specs
+An optional list of spec files can be piped to wdio that will override configured specs.
+Same applies to the exclude option. It can take a list of specs to exclude for a given run
+and it also overrides the exclude key from the config file.
 
 Commands:
   wdio.js repl <browserName>  Run WebDriver session in command line
+  wdio.js install <type> <name> Add a `reporter` or a `service` to your WebdriverIO project
```

Comment:
Should the doc itself be updated to mention `framework`?

Reply:
Yes, I have a comment [here](https://github.com/webdriverio/webdriverio/pull/4010#issuecomment-496209622) explainig this.

Reply:
Ah, got it, you are working on it, thanks